core/src/main/java/nl/nn/adapterframework/filesystem/ExchangeFileSystem.java

```diff
@@ -917,6 +908,15 @@ protected String getFrom(EmailMessage emailMessage) throws FileSystemException {
 		}
 	}
 
+	protected String getReceivedBy(EmailMessage emailMessage) throws FileSystemException {
+		try {
+			return emailMessage.getReceivedBy().getAddress();
+		} catch (ServiceLocalException e) {
+			log.warn("Could not get ReceivedBy Address: "+ e.getMessage());
+			return null;
```

Comment:
Hier niet liever FileSystemException gooien?

Reply:
Ik heb de getFrom en getSender eigenlijk overgenomen qua methode structuur, hier worden ook geen FileSystemExceptions gegooid op het moment dat er een ServiceLocalException optreedt.

Ik kan het toevoegen, maar moet het dan daar ook niet toegevoegd worden?

Reply:
getFrom en getSender voorlsnog niet aanpassen. 
Hier wel, of anders een null-check doen als je de waarde gebruikt.